ownCloud/Client/ClientQueryViewController.swift

```diff
@@ -425,6 +446,36 @@ extension ClientQueryViewController : SortBarDelegate {
 	func sortBar(_ sortBar: SortBar, didUpdateSortMethod: SortMethod) {
 		sortMethod = didUpdateSortMethod
 		query?.sortComparator = sortMethod.comparator()
+
+	}
+}
+
+// MARK: - UISearchResultsUpdating Delegate
+extension ClientQueryViewController: UISearchResultsUpdating {
+	func updateSearchResults(for searchController: UISearchController) {
+		let searchText = searchController.searchBar.text!
+
+		let filterHandler: OCQueryFilterHandler = { (_, _, item) -> Bool in
+			if let item = item {
+				if item.name.lowercased().contains(searchText.lowercased()) {return true}
```

Comment:
Please use `localizedCaseInsensitiveContains()` instead. We got a lot of bonus functionality for free this way. F.ex. searching for `o` should also match`ö`, `ó`, `ò` etc.

Reply:
I've tried
```Swift
if item.name.localizedCaseInsensitiveContains(searchText.localizedLowercase) {return true}
```

and 

```Swift
if item.name.localizedCaseInsensitiveContains(searchText) {return true}

```

and I can't achieve what you said:

> F.ex. searching for o should also matchö, ó, ò etc.

There is something that I'm missing?